catalyst/utils/metrics/tests/test_cmc_metric_callback.py

```diff
@@ -0,0 +1,50 @@
+import torch
```

Comment:
let's divide this test into two parts:
- cmc_score metric test
- CMCScoreCallback test